docs/_posts/2015-09-04-react-v0.14-rc1.md

```diff
@@ -0,0 +1,175 @@
+---
+title: "React v0.14 Release Candidate"
+author: spicyj
+---
+
+We're happy to announce our first release candidate for React 0.14! We gave you a [sneak peek in July](/react/blog/2015/07/03/react-v0.14-beta-1.html) at the upcoming changes but we've now stabilized the release more and we'd love for you to try it out before we release the final version.
+
+Let us know if you run into any problems by filing issues on our [GitHub repo](https://github.com/facebook/react).
+
+## Installation
+
+We recommend using React from `npm` using a tool like browserify or webpack to build your code into a single package:
+
+* `npm install --save react@0.14.0-rc1`
+* `npm install --save react-dom@0.14.0-rc1`
+
+Remember that by default, React runs extra checks and provides helpful warnings in development mode. When deploying your app, set the `NODE_ENV` environment variable to `production` to use the production build of React which does not include the development warnings and runs significantly faster.
+
+We also provide pre-built browser builds for your convenience:
+
+* **React**  
+  Dev build with warnings: <https://fb.me/react-0.14.0-rc1.js>  
+  Minified build for production: <https://fb.me/react-0.14.0-rc1.min.js>  
+* **React with Add-Ons**  
+  Dev build with warnings: <https://fb.me/react-with-addons-0.14.0-rc1.js>  
+  Minified build for production: <https://fb.me/react-with-addons-0.14.0-rc1.min.js>  
+* **React DOM** (include React in the page before React DOM)  
+  Dev build with warnings: <https://fb.me/react-dom-0.14.0-rc1.js>  
+  Minified build for production: <https://fb.me/react-dom-0.14.0-rc1.min.js>  
+
+We've also published these files in the `react` and `react-dom` packages on bower.
+
+## Changelog
+
+### Major changes
+
+- **Two Packages**
+
+    As we look at packages like [react-native](https://github.com/facebook/react-native), [react-art](https://github.com/reactjs/react-art), [react-canvas](https://github.com/Flipboard/react-canvas), and [react-three](https://github.com/Izzimach/react-three), it's become clear that the beauty and essence of React has nothing to do with browsers or the DOM.
```

Comment:
Oh that's a giant list… that was hard to see in the diff. I guess that's ok (and then you can ignore the `h4` comment).